CoffeeChat/Controller/SocialMediaViewController.swift

```diff
@@ -0,0 +1,202 @@
+        view.addSubview(instagramTextField)
+
+        facebookTextField.backgroundColor = .backgroundWhite
+        facebookTextField.textColor = .black
+        facebookTextField.font = ._20CircularStdBook
+        facebookTextField.placeholder = "Facebook profile link"
+        facebookTextField.clearButtonMode = .never
+        facebookTextField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingChanged)
+        let facebookImageView = UIImageView(image: UIImage(named: "facebook"))
+        facebookImageView.frame = CGRect(x: 15, y: 15, width: 19, height: 19)
+        facebookTextField.addSubview(facebookImageView)
+        facebookTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 44, height: 49))
+        facebookTextField.leftViewMode = .always
+        facebookTextField.layer.cornerRadius = 8
+        facebookTextField.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.15).cgColor
+        facebookTextField.layer.shadowOpacity = 1
+        facebookTextField.layer.shadowRadius = 4
+        facebookTextField.layer.shadowOffset = CGSize(width: 0, height: 2)
+        view.addSubview(facebookTextField)
+
+        disclaimerLabel.text = "Your information will only be shared with the verified Cornell students you are paired with."
+        disclaimerLabel.textColor = .greenGray
+        disclaimerLabel.font = ._12CircularStdBook
+        disclaimerLabel.numberOfLines = 0
+        disclaimerLabel.textAlignment = .center
+        disclaimerLabel.lineBreakMode = .byWordWrapping
+        view.addSubview(disclaimerLabel)
+
+        nextButton.setTitle("Ready for Pear", for: .normal)
+        nextButton.setTitleColor(.white, for: .normal)
+        nextButton.titleLabel?.font = ._20CircularStdBold
+        nextButton.backgroundColor = .inactiveGreen
+        nextButton.layer.cornerRadius = 26
+        nextButton.isEnabled = false
+        nextButton.addTarget(self, action: #selector(nextButtonPressed), for: .touchUpInside)
+        view.addSubview(nextButton)
+
+        skipButton.titleLabel?.font = ._16CircularStdMedium
+        skipButton.setTitle("Skip", for: .normal)
+        skipButton.setTitleColor(.greenGray, for: .normal)
+        skipButton.backgroundColor = .none
+        skipButton.addTarget(self, action: #selector(skipButtonPressed), for: .touchUpInside)
+        view.addSubview(skipButton)
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+
+        backButton.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel).offset(6)
+            make.size.equalTo(Constants.Onboarding.backButtonSize)
+            make.leading.equalToSuperview().offset(24)
+        }
+
+        titleLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.size.equalTo(CGSize(width: 318, height: 61))
+            make.top.equalTo(view.safeAreaLayoutGuide).offset(Constants.Onboarding.titleLabelPadding)
```

Comment:
Hard for me to tell exactly, but this feels like it goes beyond the usual line limit. Maybe split into

    make.top
        .equalTo(view.safeAreaLayoutGuide)
        .offset(Constants.Onboarding.titleLabelPadding)